content/jobs.html

```diff
@@ -6,12 +6,113 @@
 
 <div class="col-sm-8">
   <h2>Jobs</h2>
+  <p>
+    The MapLibre Organization is an umbrella for open-source mapping libraries
```

Comment:
You can create shortcodes out of both possibilities: https://gohugo.io/content-management/shortcodes/

Then it might be simpler to selectivly call on of both shortcodes.

Reply:
I don't understand, can you elaborate?